packages/flame_jenny/example/.gitignore

```diff
@@ -0,0 +1,43 @@
+# Miscellaneous
```

Comment:
This file should not be included, there is already a gitignore file in the root of the monorepo handling it.

Reply:
Removed in [bbf0135](https://github.com/flame-engine/flame/pull/3086/commits/bbf01353a5a89312cd105136eabbf20a2e366a7d)